pages/_document.tsx

```diff
@@ -44,6 +45,18 @@ export default class MyDocument extends Document {
               `,
             }}
           />
+          {/* Microsoft Clarity */}
+          <script
+            dangerouslySetInnerHTML={{
+              __html: `
+                (function(c,l,a,r,i,t,y){
+                  c[a]=c[a]||function(){(c[a].q=c[a].q||[]).push(arguments)};
+                  t=l.createElement(r);t.async=1;t.src="https://www.clarity.ms/tag/"+i;
+                  y=l.getElementsByTagName(r)[0];y.parentNode.insertBefore(t,y);
+                })(window, document, "clarity", "script", "pepjushhm5");
+              `,
+            }}
+          />
```

Comment:
Wonder if we can do it like with Hotjar – https://github.com/tinacms/tina.io/blob/main/pages/_app.tsx (see the useEffect)

Reply:
Also should we remove Hotjar if we're moving to MS clarity.

Reply:
Done - we're keeping both HotJar and Clarity to compare coverage for now